src/query.ts

```diff
@@ -138,7 +163,7 @@ export class Query {
    * @param filterName  The name of the filter to modify
    * @param value       The value to remove from the filter
    */
-  public removeExcludedFilter(filterName: string, value: AssignFilterValue): Query {
+  public removeExcludedFilter(filterName: string, value: InputFilterValue): Query {
```

Comment:
Rename this one: `removeExcludedFilter => removeExcludeFilter` or rename `addExcludeFilter => addExcludedFilter` (`Exclude => Excluded`)

Reply:
done